packages/console/src/utilities/translation.ts

```diff
@@ -0,0 +1,9 @@
+import i18next from 'i18next';
+
+export const translateUnnamed = (resource: Record<string, string>) => {
+  const { languages } = i18next;
+  const matchedLanguage = languages.find((language) => resource[language]);
+
+  // Note: the `Unknown` string will not be returned in theory.
+  return (matchedLanguage && resource[matchedLanguage]) ?? 'Unknown';
```

Comment:
just return undefined?

Reply:
I want to keep it returning `Unknown`, for we don't need to fall back on the return value whenever we use this method and want to get a `string!` value, and also, this method will return `string!` type value in theory.
e.g.:
```js
const result: string = translateUnnamed(resource) ?? 'Unknown';
```

Reply:
I would prefer either to return null as before or return some raw data from the resource. i.e. first non-empty key.  'Unknow' makes no difference compared with null, as the user won't get any info from it.  Besides. some zh chars + 'Unknown' is weird as well;

Reply:
Or just return undefined, and let the caller handle the fallback logic.  i.e. if no connector name found, return the connector target.

Reply:
Make sense.

Reply:
updated.